### PR TITLE
Clarify experimental status

### PR DESCRIPTION
**Pull request by rhetoric101**
This is a formal request to clarify the experimental status of the Elixir agent. I tried to follow some of the conventions in this template for experimental projects: https://github.com/newrelic/newrelic-experimental-repo-template/tree/main?tab=readme-ov-file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+We encourage your contributions to improve [project name]! Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project. If you have any questions, or to execute our corporate CLA, required if your contribution is on behalf of a company, please drop us an email at [opensource@newrelic.com](mailto:opensource@newrelic.com).
+
+A note about vulnerabilities: As noted in our security policy, New Relic is committed to the privacy and security of our customers and their data. We believe that providing coordinated disclosure by security researchers and engaging with the security community are important means to achieve our security goals.
+
+If you believe you have found a security vulnerability in this project or any of New Relic's products or websites, we welcome and greatly appreciate you reporting it to New Relic through HackerOne.
+
+### License
+The open-source Elixir agent is licensed under the Apache 2.0 License.
```
Comment (rhetoric101) [inline README.md:27]: Is this license section correct?

**Review by XiXiaPdx — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (XiXiaPdx) [reply README.md:27]: seems so

**Comment by rhetoric101** [comment]
Thanks, @XiXiaPdx!

**Pull request merged**